test/src/unit/modules/metricsService.test.js

```diff
@@ -206,18 +204,18 @@ describe('metricsService/index.js', () => {
                     fs.outputFileSync(pathToJvmOptions, expectedNewJvmOptions);
                     fs.removeSync(pathToBackupJvmOptions);
                 });
-                it(`throws a useful error`, async() => {
-                    await metricsService.injectMetricsCollectorIntoProject('openLiberty', projectDir)
-                        .should.be.eventually.rejectedWith('project files already backed up (i.e. we have already injected metrics collector');
-
-                    fs.readFileSync(pathToJvmOptions, 'utf8')
-                        .should.equal(expectedNewJvmOptions);
-                    fs.pathExistsSync(pathToBackupJvmOptions).should.be.false;
-
-                    (await readXml(pathToTestPomXml))
-                        .should.deep.equal(originalPomXmlForOpenLiberty);
-                    (await readXml(pathToTestPomXml))
-                        .should.deep.equal(originalPomXmlForOpenLiberty);
+                it(`should re-inject without error`, async() => {
+                    await metricsService.injectMetricsCollectorIntoProject('docker', 'java', projectDir);
+
+                    const outputJvmOptions = fs.readFileSync(pathToJvmOptions, 'utf8');
+                    outputJvmOptions.should.equal(expectedNewJvmOptions);
+                    const outputBackupJvmOptions = fs.pathExistsSync(pathToBackupJvmOptions);
```

Comment:
nit: `outputJvmOptions` is the contents of a jvm.options file, yet `outputBackupJvmOptions` is a boolean. Maybe rename the latter